tests/Ltb/LanguageTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+require __DIR__ . '/../../vendor/autoload.php';
+
+use PHPUnit\Framework\TestCase;
+
+final class LanguageTest extends \Mockery\Adapter\Phpunit\MockeryTestCase
+{
+    function test_accept_all_language() {
+        # User-Agent Language
+        $_SERVER['HTTP_ACCEPT_LANGUAGE'] = "ar;q=1, cn;q=1, de;q=1, el;q=1, es;q=1, fr;q=1, it;q=1, nb-NO;q=1, pl;q=1, pt-PT;q=1, ru;q=1, sl;q=1, tr;q=1, zh-CN;q=1, ca;q=1, cs;q=1, ee;q=1, en;q=1, eu;q=1, hu;q=1, ja;q=1, nl;q=1, pt-BR;q=1, rs;q=1, sk;q=1, sv;q=1, uk;q=1, zh-TW;q=1";
+        
```

Comment:
please remove all extra spaces at the end of lines

Reply:
Removed the long list of languages as it is an unrealistic representation of a User-Agent Accept-Language variable.

Reply:
Ok for your last change, but please also remove extra spaces at the end of lines.